plugins/vlc-video/vlc-video-source.c

```diff
@@ -303,7 +303,7 @@ static unsigned vlcs_video_format(void **p_data, char *chroma, unsigned *width,
 
 	new_format = convert_vlc_video_format(chroma, &new_range);
 
-	libvlc_video_get_size_(c->media_player, 0, width, height);
+	//int res = libvlc_video_get_size_(c->media_player, 0, width, height);
```

Comment:
For streams this will not give the right frame size.
The arguments are already set, why is this needed anyways?